openqa-label-known-issues

```diff
@@ -143,7 +136,17 @@ investigate_issue() {
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
-        handle_unreachable_or_no_log "$id" "$testurl" "$out"
+        handle_unreachable "$testurl" "$out"
+        # Assume autoinst-log.txt still not found but process job anyway due to reason var
+        if [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]]; then
+            label_on_issues_from_issue_tracker "$id"
```

Comment:
For consistency this could be

```suggestion
            label_on_issues_from_issue_tracker "$id"
            return
```

However in practice this does not matter as we don't do any other action within `investigate_issue` after here anyway.